src/Pyz/Yves/ProductDetailPage/ProductDetailPageDependencyProvider.php

```diff
@@ -35,7 +35,7 @@ public function provideDependencies(Container $container): Container
      *
      * @return \Spryker\Yves\Kernel\Container
      */
-    protected function addProductStorageClient(Container $container): Container
+    protected function addProductStoragePyzClient(Container $container): Container
```

Comment:
Pyz is not used in other places in the demoshop, please revert (I guess this is not the reason of the bug)

Reply:
This is exactly the reason of the bug because the method with the previous name overrides the core method which leads to missing CLIENT_PRODUCT_STORAGE container key.
Another one fix in the template is fixing another one bug that appeared after the mentioned one